test/ClangImporter/unserializable-clang-function-types.swift

```diff
@@ -0,0 +1,6 @@
+// RUN: %target-swift-frontend -typecheck -swift-version 5 -emit-module-interface-path - -sdk %clang-importer-sdk -enable-library-evolution %s -experimental-print-full-convention -verify
+
+import ctypes
+
+public var f1 : UnserializableFunctionPointer?
+// expected-error@-1 {{cannot export the underlying C type of the function type 'UnserializableFunctionPointer' (aka '@convention(c) () -> Optional<OpaquePointer>'); it may use anonymous types or types defined outside of a module}}
```

Comment:
Is this theoretically a source break, because people might have code that is exposing opaque/non-modular types publicly in Swift? If yes, do you think this is unlikely to happen in practice?

Reply:
It is theoretically a source break, but I wouldn't expect it to be a problem in practice as long as we catch the major cases of unimportable types.